ros/lib/host_inventory.py

```diff
@@ -0,0 +1,10 @@
+import requests
+from ros.config import INVENTORY_URL
+
+
+def fetch_host_from_inventory(insights_id, rh_identity):
+    host_api_url = f"{INVENTORY_URL}/api/inventory/v1/hosts?insights_id={insights_id}"
+    headers = {'x-rh-identity': rh_identity, 'Content-Type': 'application/json'}
+    res = requests.get(host_api_url, headers=headers)
+    hosts = res.json()
+    return hosts
```

Comment:
Also, can we rename this file to `host_inventory_interface.py` something like that? Just to get a better understanding  of what this file might contain.